applications/launchpad_v2/src/containers/WalletContainer/TariWallet/TariId.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react'
+import { useTheme } from 'styled-components'
+
+import Smiley from './Smiley'
+import { Label, SemiTransparent, TariIdContainer, TariIdBox } from './styles'
+
+const TariId = ({
+  tariId,
+  emojiTariId,
+}: {
+  tariId: string
+  emojiTariId: string[]
+}) => {
+  const [showEmoji, setShowEmoji] = useState(false)
+  const theme = useTheme()
+
+  return (
+    <>
+      <Label>
+        Tari Wallet ID <SemiTransparent>(address)</SemiTransparent>
+      </Label>
+      <TariIdContainer>
+        <TariIdBox>{showEmoji ? emojiTariId.join(' | ') : tariId}</TariIdBox>
+        <div
```

Comment:
NIT: not sure if it will make any difference, but maybe it should be a `button` instead of `div`

Reply:
it's going to be better semantically, will change